### PR TITLE
GitHub templates

### PR DESCRIPTION
**Pull request by b00f**
- Adding templates for Issues
- Adding template for Pull Requests
- Adding Contributing doc

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Contributing to Zarb project
+
+A warm welcome for considering contributing to Zarb project!
+
+## Getting Started
+
+Contributions are made to this repo via Issues and Pull Requests (PRs).
+
+<TO_BE_DEFINED>
```
Comment (d4rk5h0t) [inline CONTRIBUTING.md:9]: might be a good idea to link discord channel here

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+> Detail the steps taken to reproduce this error, what was expected, and whether this issue can be reproduced consistently or if it is intermittent.
+>
+> Please include any log information if you have.
+
+### Environment
+
+> Please provide the following:
+
+- **Operation System and Version**
+- **Zarb Version**
```
[the remark this answers is not in the document]
Comment (b00f) [reply .github/ISSUE_TEMPLATE/bug_report.md:33]: Zarb inside docker should acts same as binary files. I think that is not necessary.

File: CONTRIBUTING.md (hunk 1 above)
Comment (b00f) [reply CONTRIBUTING.md:9]: @loreanvictor What is your idea?

Comment (loreanvictor) [reply CONTRIBUTING.md:9]: templates look good. CONTRIBUTING.md of course needs work, I would recommend outlining project structure, build and testing process (so people can also run tests on their own dev environments), and code conventions that people should follow (basically stuff that you would need to explain on a per-PR basis, if this file did not exist).

**Review by d4rk5h0t — APPROVED** [review]
all look good to me just we got [ZIP Repo](https://github.com/zarbchain/zips) to keep track of Requested Features better i think we better link it in `feature_request.md`

**Comment by b00f** [comment]
> all look good to me just we got ZIP Repo to keep track of Requested Features better i think we better link it in feature_request.md

Good point. We can mention it.

**Comment by b00f** [comment]
I remove contribute guideline from this PR and opened #187

**Pull request merged**